web/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
This react logo svg can prob be deleted since it's not used anywhere besides the browser tab icon